package.json

```diff
@@ -10,6 +10,7 @@
 		"build:server": "pnpm nx build @overture-stack/lectern-server",
 		"build:client": "pnpm nx build @overture-stack/lectern-client",
 		"test:all": "pnpm nx run-many --all --target=test",
+		"integration-test:all": "pnpm nx run-many --all --target=integration-test",
```

Comment:
How about using three scripts:
- test:unit -- runs unit tests
- test:integration -- runs integration tests, may include testcontainers
- test -- runs test:unit and test:integration. Can also have `test:all` which `test` would be an alias of.

Reply:
As there are multiple projects renamed scripts to include the package name. see [comment](https://github.com/overture-stack/lectern/pull/231#issuecomment-2377811701)